.github/workflows/docker-image.yml

```diff
@@ -29,4 +29,4 @@ jobs:
           tags: |
             ghcr.io/${{ github.repository }}/rust-musl-builder:1.64.0-llvm-cov
```

Comment:
@fosskers You have to change tag 1.64 to 1.65.

Reply:
Fixed. Thanks, I should have been more careful.